test/test_dynamic_shapes.py

```diff
@@ -204,7 +205,7 @@ def test_symint_args(self):
         y = create_symbolic_tensor("y", torch.randn(5, 4, 1), shape_env)
         LAST_DIM = 2
         z = x.narrow_copy(LAST_DIM, 0, y.shape[LAST_DIM])
-        self.assertTrue(z.shape[2] == int(y.shape[2]))
```

Comment:
why this change?

Reply:
We used to return integers sometimes, but now we consistently return SymIntNodes (which may wrap integers). I think this is probably better, but open to suggestions.